src/pudl/glue/ferc1_eia.py

```diff
@@ -203,7 +208,7 @@ def transform_main(self, df: pd.DataFrame) -> pd.DataFrame:
         )
 
 
-def get_raw_plants_ferc1(
+def get_plants_ferc1_raw(
```

Comment:
I could be wrong here, but I think that **all** of the `Ferc1Settings` objects which are instantiated in here should probably use the same parameters, including the `years`, which isn't currently used for creating the settings object that gets used (Though it is done on the first line of the function where the years are validated. Actually I don't think this validation needs to be done any more since Pydantic will always validate the years).

Reply:
good point. updated!